config/connect-log4j.properties

```diff
@@ -13,13 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-log4j.rootLogger=INFO, stdout
+log4j.rootLogger=INFO, stdout, ROLLINGFILE
```

Comment:
We should follow the conventions in the `log4j.properties` file in the same directory, and use a logical name in camelcase (e.g., `connectAppender`).